files/scripts/profiling/deployment_time_profiling.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import pandas as pd
+from datetime import datetime
+import sys
+
+def help():
+    print("\nThe script requires exactly 2 arguments in sequence\n1.Input file\n2.Output file")
+    print("\nSample command:\n $ python time-profile-script.py input output")
+args = sys.argv
+if len(args) == 2 and (args[1] == '-h' or args[1] == '--help'):
+    help()
+    exit()
```

Comment:
If it's a non-standard exit, the function should be passed an appropriate argument.
Else, `exit()` call is unnecessary.